cli/command/format.go

```diff
@@ -131,6 +134,7 @@ func genFormatPlaybook(curveadm *cli.CurveAdm,
 	stopFormat := options.stopFormat
 	debug := options.debug
 	clean := options.clean
+	increment := options.increment
 
 	steps := FORMAT_PLAYBOOK_STEPS
 	if showStatus {
```

Comment:
may be we can use playbook `playbook.STOP_SERVICE`.
for example:

```go
 FORMAT_INCREMENT_PLAYBOOK_STEPS := int[]{
  playbook.STOP_SERVICE,
  playbook.FORMAT_CHUNKFILE_POOL,
}

if increment {
  steps = FORMAT_INCREMENT_PLAYBOOK_STEPS
}

stopChunkServercs = curveadm.FilterDeployConfig(dcs, topology.FilterOption{
		Role: "chunkserver",
})
if len(stopChunkServercs) == 0 {
	// no chunkserver
}
if step == playbook.STOP_SERVICE {
     pb.AddStep(&playbook.PlaybookStep{
        ...
        Configs: stopChunkServercs
     } else ...
}
```

Reply:
done